tests/src/Traits/ExampleMediaTrait.php

```diff
@@ -0,0 +1,94 @@
+<?php
```

Comment:
I don't like some of this stuff that much, as I suspected in 2020 there is and will be a need for these helper methods
https://github.com/openeuropa/oe_media/pull/124/files
I think this is rather the way to go, but I don't know if we can get that merged in or not

Reply:
@brummbar what do you think ?

Reply:
I don't like the idea of putting a dependency on test code coming from another module. I don't even much like the trait in se, we use it in one class only, we could have created simple protected methods.
Let's avoid to have to deal with oe_media too, let's put back the code inside our test class. We will move it when we need it.